ozaria/site/styles/play/level/goals.sass

```diff
@@ -3,11 +3,10 @@
 
 #goals-view
   height: 145px
-  $width: 528px
+  $width: 100%
```

Comment:
`$width`?

Reply:
Thank you for asking this! It was a remnant from a previous iteration of this code, and is no longer in use. Removing 👍